File: 1184. Distance Between Bus Stops.cpp
class Solution {
public:
    int distanceBetweenBusStops(vector<int>& a, int x, int y) {
        if (x > y) swap(x, y);
        int s = accumulate(a.begin(), a.end(), 0);
        int d = accumulate(a.begin()+x, a.begin()+y, 0);
        return min(d, s-d);
    }
};

File: 1184. Distance Between Bus Stops.py
class Solution:
    def distanceBetweenBusStops(self, a: List[int], x: int, y: int) -> int:
        if x > y:
            x, y = y, x
        s, d = sum(a), sum(a[x:y])
        return min(d, s-d)